Tests/ModelObjects.swift

```diff
@@ -49,7 +49,7 @@ func ==(lhs: TestObject, rhs: TestObject) -> Bool {
 
 // MARK: -
 
-extension TestObject: Encodable, Decodable {
+extension TestObject: Elevate.Encodable, Elevate.Decodable {
```

Comment:
Do you think the the example code in the README also have the `Elevate` namespace prefixed?

Reply:
Yep, good catch. Will fix. 👍🏻

Reply:
Addressed in d9c258f4.